registry/sql-registry/registry/models.py

```diff
@@ -117,6 +117,15 @@ class RelationshipType(Enum):
     Produces = 4
 
 
+    def __str__(self):
```

Comment:
This actually already exists as `Enum.name` property.

Reply:
Added this one to support str(t) at line 422. Or to modify it to t.name as you mentioned.